test/unit/packages.config

```diff
@@ -2,6 +2,6 @@
 <packages>
   <package id="Microsoft.ApplicationInsights" version="2.2.0-beta1" targetFramework="net46" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net46" />
-  <package id="Npgsql" version="3.0.5" targetFramework="net46" />
+  <package id="Npgsql" version="3.1.8" targetFramework="net46" />
```

Comment:
If you don't require this update, no need to update to newer Npgsql version here